go/client/rpc.go

```diff
@@ -145,6 +145,14 @@ func GetUserClient(g *libkb.GlobalContext) (cli keybase1.UserClient, err error)
 	return
 }
 
+func GetPhoneNumbersClient(g *libkb.GlobalContext) (cli keybase1.PhoneNumbersClient, err error) {
+	var rcli *rpc.Client
+	if rcli, _, err = GetRPCClientWithContext(g); err == nil {
+		cli = keybase1.PhoneNumbersClient{Cli: rcli}
+	}
+	return
+}
```

Comment:
man, can you imagine how neat this file could be if we had generics or any sort of templating in go? one can dream

Reply:
lol. it would not need to be a whole file.